docs/ABOUT.md

```diff
@@ -1,35 +1,7 @@
+[Java](https://go.java/index.html) is among the most popular available programming languages, thanks to its versatility and compatibility. It is widely used for software development, mobile applications, and large systems development.
 
-[Java](https://en.wikipedia.org/wiki/Java_(programming_language)) is a general-purpose computer programming language that is: 
- 1. [Concurrent](https://en.wikipedia.org/wiki/Concurrent_computing) 
- 2. [Class-Based](https://en.wikipedia.org/wiki/Class-based_programming)
- 3. [Object-Oriented](https://en.wikipedia.org/wiki/Object-oriented_programming)
+Besides it is not that young as the fresh languages out there, Java which was born in 1995 and is maintained by [Oracle](https://www.oracle.com/index.html) was designed to be fast, secure, reliable, beginner-friendly and highly portable. This portability perk exists as it must be execute through a cross-platform compatible [Java Virtual Machine - JVM](https://en.wikipedia.org/wiki/Java_virtual_machine). Android apps are also developer using Java, since the [Android Operating System](https://en.wikipedia.org/wiki/Android_(operating_system)) runs on a Java language environment.
 
-Java applications are typically compiled to [bytecode](https://en.wikipedia.org/wiki/Bytecode) that can run on any [Java virtual machine](https://en.wikipedia.org/wiki/Java_virtual_machine) (JVM) regardless of computer architecture.
+The Java community size is huge (GitHub for example has over 1.5 million Java projects, also mentioning that Java has the second larger community in [StackOverflow](https://stackoverflow.com/questions/tagged/java))! This is important because the larger a programming language community is, the more support you'd be likely to get.
```

Comment:
This also doesn't really make sense in English. You could slightly rephrase it to:
"
The Java community is huge. GitHub for example has over 1.5 million Java projects. It's also worth mentioning that Java has the second largest community in [StackOverflow](https://stackoverflow.com/questions/tagged/java)! This is important because the larger a programming language community is, the more support you'd be likely to get.
"